Documentation/getting-started.md

```diff
@@ -0,0 +1,134 @@
+# Getting Started with acbuild
+
```

Comment:
acbuild aims to support a workflow very similar to `Dockerfile`s, but with more flexibility and adherence to the Unix tools philosophy, and native image output in the modern ACI format. Generally, the workflow consists of a shell script (analogous to the `Dockerfile`) driving acbuild to construct an ACI, either from scatch or with another image as a base. This ACI is then stored in an *image registry*, and fetched by users to execute the applications stored within.